### DISCUSSION TITLE
Welcome to Practice-Code Discussions!

### DISCUSSION THREAD

**Discussion by ShashankShishodia** [ShashankShishodia/Practice-Code#5] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShashankShishodia21** [comment, marked as answer]
i think the page should be responsive
